dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/views/StrainView.java

```diff
@@ -52,4 +59,47 @@ public void resetTranslate() {
 		super.resetTranslate();
 		updateStrain();
 	}
+	
+	/**
+	 * Sets the panning of the {@link StrainView}.
+	 * @param x the amount to pan on the x axis.
+	 * @param y the amount to pan on the y axis.
+	 */
+	public void setPan(double x, double y) {
+		resetZoom();
```

Comment:
You can restore the current zoomlevel by saving it, see [here](https://github.com/Unia/dnainator/blob/minimap/dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/views/StrainView.java#L54) in my minimap branch.